pretty/src/geometry/mod.rs

```diff
@@ -6,5 +6,4 @@ mod region;
 
 pub use self::bound::Bound;
 pub use self::pos::{Col, Pos, Row, MAX_WIDTH};
-pub use self::rect::Rect;
```

Comment:
TODO: make `Rect.body` and `Rect.last_line` private, now that they're once again not used outside of `Rect`.